packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1157,47 +1157,53 @@ export default class SalesforceCRMService implements CRM {
       }
 
       // Handle different field types
-      if (fieldConfig.fieldType === field.type) {
```

Comment:
Removing this line since the `fieldConfig.fieldType` might equal `string` but we want to reuse the logic for different types of fields.